packages/design-system/src/__generated__/figma-design-tokens.ts

```diff
@@ -493,28 +535,31 @@ export const color = {
   foregroundOverwrittenFlexUi: "#dc3d43",
   foregroundInversePrimary: "#ff22ae",
   foregroundLinkNotVisited: "#006adc",
+  foregroundLinkHoverMain: "#656869",
+  foregroundLinkHoverTextSubtle: "#656869",
```

Comment:
Why is it called HoverText ? if you are styling a link color its always the text, why is it not "foregroundLinkHoverSubtle"

Reply:
I don't remember where this token came from. It's old, not part of this update.

Reply:
Not all links are text

Reply:
There's a color token called "foreground.text-subtle". If we have a text link that uses foreground.text-subtle, then the hover color would be foreground.link-hover.text-subtle

Reply:
there is no link with non-text, so its semantically confusing

Reply:
you can ask Mark to rename both text-subtle and the link hover to not use the word text